esp/src/src/nls/hu/hpcc.ts

```diff
@@ -580,6 +595,7 @@ export = {
     Part: "Rész",
     PartMask: "Fizikai fájl név maszk",
     PartName: "Fájl rész név",
+    PartNumber: "Rész sorszám",
```

Comment:
I think it would be better "Fájl rész sorszám"

Or it would be more consistent if:
 Part: "Fizikai fájl rész",
 PartName: "Fizikai fájl rész név",
 PartNumber: "Fizikai fájl rész sorszám",

Reply:
Change made, please review @AttilaVamos